deployment-guides/AWS-EKS-EFS/deploy_efs_shared_fs.sh

```diff
@@ -2,18 +2,18 @@
 
 echo "1. Creating EFS file system"
 
-aws efs create-file-system --creation-token "$FILESYSTEM_NAME" --performance-mode generalPurpose --throughput-mode bursting --region $AWS_REGION --tags Key=Name,Value="Galaxy filesystem" Key=developer,Value="$DEVELOPER"
+aws efs create-file-system --creation-token "$FILESYSTEM_NAME" --performance-mode maxIO --throughput-mode bursting --region $AWS_REGION --tags Key=Name,Value="Galaxy filesystem" Key=developer,Value="$DEVELOPER"
 fs_id=$(aws efs --region $AWS_REGION describe-file-systems --creation-token $FILESYSTEM_NAME --query "FileSystems[0].FileSystemId" --output text)
 
 echo -e "Created file system ID $fs_id"
 
 # Derive networking info from the cluster
 
-vpc_id=$(aws ec2 describe-vpcs --output text --region $AWS_REGION --filters "Name=tag:Name,Values=eksctl-${CLUSTER_NAME}-cluster/VPC" --query "Vpcs[0].VpcId")
+vpc_id=$(aws ec2 describe-vpcs --output text --region $AWS_REGION --filters --filters "Name=isDefault,Values=true" --query "Vpcs[0].VpcId" --query "Vpcs[0].VpcId")
```

Comment:
```suggestion
vpc_id=$(aws ec2 describe-vpcs --output text --region $AWS_REGION --filters "Name=isDefault,Values=true" --query "Vpcs[0].VpcId" --query "Vpcs[0].VpcId")
```

removing extra --filters that I presume are not needed.

Reply:
I have changed the filter to fetch information about the default VPC. the original command was always failing to get the right VPC information to use with EFS. I have opted to use the default VPC used by the EC2 nodes, as it is already configured with the right subnets and elastic IPs to enable a smooth setup for the EKS nodes and load balancer. Creating, configuring and testing a new VPC with all the required configurations at the time was very time consuming with my limited knowledge at the time. I can add setting up and configuration of a new VPC later if this is a much-needed task.